themes/vocabulary_theme/templates/blocks/featured-projects.html

```diff
@@ -0,0 +1,46 @@
+<section class="featured-projects">
+  <div class="container">
+    <div class="level">
+      <h4 class="is-paddingless level-left">
+        {{ this.title }}
+      </h4>
+      <span class="level-right">
+        <a class="project-page-link" href="/contributing-code/projects/">See all available projects <i class="icon angle-right"></i></a>
+      </span>
+    </div>
+    <div class="project-list columns">
+      {% for repo in bag('repos.repos') %}
+      {% if repo.featured %}
+        <article class="card entry-post vertical project-index column is-one-third">
+          <div class="card-content">
+            <h4 class="card-title b-header"><a href="{{ repo.url }}">{{ repo.english_name }}</a></h4>
+            <a class="button is-text tiny site-link" href="{{ repo.website }}">
+              <span>Visit site</span>
+              <i class="icon external-link"></i>
+            </a>
+            <div class="content">
+              {{ repo.description }}
+            </div>
+            <div class="labels">
```

Comment:
I think you need to add a `margin-bottom` on tag buttons 
![Screen Shot 2020-07-06 at 4 35 25 PM](https://user-images.githubusercontent.com/894708/86639696-51245280-bfa7-11ea-9e0e-1c4a7243f881.png)

Reply:
I have opened an issue regarding this in vocabulary itself - [#499](https://github.com/creativecommons/vocabulary/issues/499)